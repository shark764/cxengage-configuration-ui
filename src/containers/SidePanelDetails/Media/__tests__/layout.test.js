/*
 * Copyright Â© 2015-2019 Serenova, LLC. All rights reserved.
 */

import React from 'react';
import { shallow } from 'enzyme';

import MediaDetailsPanel from '../layout';

describe('<MediaDetailsPanel />', () => {
  it('renders media detailsPanel', () => {
    const rendered = shallow(<MediaDetailsPanel className="details-panel" children={'Mock Child'} />);
    expect(rendered).toMatchSnapshot();
  });
});
