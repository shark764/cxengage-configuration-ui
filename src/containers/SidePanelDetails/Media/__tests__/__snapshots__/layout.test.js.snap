// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MediaDetailsPanel /> renders media detailsPanel 1`] = `
<styled.div>
  Mock Child
</styled.div>
`;
